internal/dinosaur/pkg/api/admin/private/model_central_request.go

```diff
@@ -27,7 +27,8 @@ type CentralRequest struct {
 	Region       string    `json:"region,omitempty"`
 	Owner        string    `json:"owner,omitempty"`
 	Name         string    `json:"name,omitempty"`
-	Host         string    `json:"host,omitempty"`
+	UiHost       string    `json:"ui_host,omitempty"`
+	DataHost     string    `json:"data_host,omitempty"`
```

Comment:
WDYTH about `SensorHost` instead? This would reduce the mapping overhead for this route

```suggestion
	SensorHost     string    `json:"data_host,omitempty"`
```

Reply:
I think it will cause a confusion because we're already using `dataHost` in a few places in the code, for example in a private API.
`data` term is also mentioned in the hostname (agreed in the ADR), and on the design presentation. So I'd like to keep is as is.